src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -1079,6 +1079,38 @@ export default class CommandControl extends React.Component {
 		this.missionHistory.push(deepcopy(mission))
 	}
 
+	/**
+	 * Used to differentiate between a state change and a task/mission change. Called from onDoneClick() in Render(). Calls updateMissionHistory() if the task of a waypoint changes. 
```

Comment:
What do you think of reducing this comment to just the last sentence?

Reply:
I like it, done!